app/src/forms/attestations/emailService.js

```diff
@@ -0,0 +1,251 @@
+    return [
+      {
+        context: {
+          confirmationNumber: submission.confirmationId,
+          business: submission.business.name,
+          status: statusCode.display,
+          message: settings.config.message,
+          title: settings.config.title,
+          messageLinkText: settings.config.messageLinkText,
+          messageLinkUrl: `${settings.config.messageLinkUrl}/${submission.submissionId}`
+        },
+        to: [statusUpdate.assignedToEmail]
+      }
+    ];
+  }
+
+  async sendStatusAssignmentEmail(statusUpdate) {
+    try {
+      const settings = await this._dataService.readSettings('statusAssignmentEmail');
+      if (settings && settings.enabled && statusUpdate.assignedToEmail) {
+        if (!this._statusAssignmentEmailBody) {
+          this._statusAssignmentEmailBody = fs.readFileSync(`${this._assetsPath}/${settings.config.template}`, 'utf8');
+        }
+
+        const contexts = await this._getStatusAssignmentEmailContexts(settings, statusUpdate);
+
+        const data = {
+          body: this._statusAssignmentEmailBody,
+          bodyType: 'html',
+          contexts: contexts,
+          ...settings.config
+        };
+        return await chesService.merge(data);
+      } else {
+        return false;
+      }
+    } catch (err) {
+      log.error('sendStatusAssignmentEmail', `Error: ${err.message}`);
+      log.error(err);
+      throw err;
+    }
+  }
+
+  async _getAccessRequestedEmailContexts(settings, accessRequest, to) {
+    return [
+      {
+        context: {
+          title: settings.config.title,
+          userInfo: `${accessRequest.firstName} ${accessRequest.lastName} - ${accessRequest.email}`,
+          message: settings.config.message,
+          messageLinkText: settings.config.messageLinkText,
+          messageLinkUrl: `${settings.config.messageLinkUrl}`
+        },
+        to: to
+      }
+    ];
+  }
+
+  async sendAccessRequestedEmail(accessRequest) {
+    try {
+      const settings = await this._dataService.readSettings('accessRequestedEmail');
+      if (settings && settings.enabled) {
+        if (!this._accessRequestedEmailBody) {
+          this._accessRequestedEmailBody = fs.readFileSync(`${this._assetsPath}/${settings.config.template}`, 'utf8');
+        }
+        const to = settings.config.to.split(',').filter(x => x);
+
+        const contexts = await this._getAccessRequestedEmailContexts(settings, accessRequest, to);
+
+        const data = {
+          body: this._accessRequestedEmailBody,
+          bodyType: 'html',
+          contexts: contexts,
+          ...settings.config
+        };
+        return await chesService.merge(data);
+      } else {
+        return false;
+      }
+    } catch (err) {
+      log.error('sendAccessRequestedEmail', `Error: ${err.message}.`);
+      log.error(err);
+      throw err;
+    }
+  }
+
+}
+
+class OperationTypesEmailService extends EmailService {
```

Comment:
Might have already thought it through, but is this class necessary, seems to just use the same email body fxns, but with an additional field I think? Could it be handled with just a conditional in the base one? Or some soread operator mixing in additional fields or something....? I don't have a solution off the top of my head but maybe something to think about?

Reply:
The class is necessary, keeps the edge cases where they should be.  But cleaned up the logic to use the parent class and add to the result, not duplicate the result with extra.